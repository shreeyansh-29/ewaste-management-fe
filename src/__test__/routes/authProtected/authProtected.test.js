/* eslint-disable no-undef */
import React from "react";
import Enzyme, {shallow} from "enzyme";
import Adatper from "@wojtekmaj/enzyme-adapter-react-17";
import AuthProtected from "../../../routes/authProtected/authProtected";
import toJson from "enzyme-to-json";

Enzyme.configure({adapter: new Adatper()});

describe("testing authProtected", () => {
  const wrapper = shallow(<AuthProtected />);
  it("test", () => {
    expect.assertions(1);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
