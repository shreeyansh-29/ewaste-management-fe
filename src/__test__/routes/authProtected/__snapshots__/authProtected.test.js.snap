// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing authProtected test 1`] = `
<Fragment>
  <Navigate
    to="/404"
  />
  <Navigate
    to="/404"
  />
  <Navigate
    to="/404"
  />
</Fragment>
`;
